include/aos/common/tools/log.hpp

```diff
@@ -195,20 +195,19 @@ class Log : private NonCopyable {
 
     Log& operator<<(const Error& err)
     {
-        *this << err.Message();
+        StaticString<cMaxErrorStrLen> tmpStr;
 
-        if (err.Errno() != 0) {
-            *this << " [" << err.Errno() << "]";
-        }
-
-        if (err.FileName() != nullptr) {
-            *this << " (" << err.FileName() << ":" << err.LineNumber() << ")";
+        if (tmpStr.Convert(err).IsNone()) {
+            return *this << tmpStr;
         }
 
         return *this;
     }
 
 private:
+    // AOS_CONFIG_ERROR_STR_LEN
+    constexpr static auto cMaxErrorStrLen = AOS_CONFIG_ERROR_STR_LEN;
```

Comment:
please move cMaxErrorStrLen to  string header.

Reply:
Done. move to error.hpp